react/features/notifications/middleware.js

```diff
@@ -69,11 +69,6 @@ MiddlewareRegistry.register(store => next => action => {
         return next(action);
     }
     case PARTICIPANT_UPDATED: {
-        if (typeof interfaceConfig === 'undefined') {
-            // Do not show the notification for mobile and also when the focus indicator is disabled.
```

Comment:
Why?

Reply:
Because we want to show the notification even on native

Reply:
That will be very disruptive because there is no config option to disable it. I suggest you migrate `DISABLE_FOCUS_INDICATOR` from interface_config.js to config.js as part of this effort so it can be easily disabled when needed.

Reply:
I can add back to check for disable_focus_indicator, but wouldn't migrating it be disruptive for the costumers that already use it? Because I can see it is whitelisted, so I assume some customers could be using it at the moment

Reply:
We have a way of retaining backwards compatibility, check here: https://github.com/jitsi/jitsi-meet/blob/58d98ad7d374632c7c06161efd04ef3a3ba9602c/react/features/base/config/reducer.js#L189

Reply:
I didn't know about that. Thanks for the info. I'll migrate it to config then